app/helpers/report_helper/editor.rb

```diff
@@ -0,0 +1,12 @@
+module ReportHelper::Editor
+  def cb_entities_by_provider_id(provider_id, entity_type)
+    provider = ManageIQ::Providers::ContainerManager.find(provider_id)
```

Comment:
Did we just introduce an N+1 for looking up providers?

I guess if we were really concerned about this, we would use `ContainerProject.where(:ems_id => provider_id)...` - which is overkill at the moment.

The number of reduced queries for `container_projects` and `container_images` has been reduced, so I'm good with the change